sw/device/lib/dif/dif_spi_device.h

```diff
@@ -105,6 +105,17 @@ OT_WARN_UNUSED_RESULT
 dif_result_t dif_spi_device_init_handle(mmio_region_t base_addr,
                                         dif_spi_device_handle_t *spi);
 
+/**
+ * Initializes a SPI device handle for use.
+ *
+ * @param dt The DT handle of the spi_device peripheral.
+ * @param[out] Out param for the initialized handle.
```

Comment:
```suggestion
 * @param[out] spi Out param for the initialized handle.
```
This was also incorrect in the original `dif_spi_device_init_handle` function - can you fix it there also?

Reply:
sure